src/test/ui/unboxed-closures/unboxed-closure-sugar-wrong-number-number-type-parameters-3.stderr

```diff
@@ -1,8 +1,14 @@
-error[E0107]: wrong number of type arguments: expected 3, found 1
+error[E0107]: wrong number of type arguments for `Three`: expected 3, found 1
   --> $DIR/unboxed-closure-sugar-wrong-number-number-type-parameters-3.rs:5:16
    |
+LL | trait Three<A,B,C> { fn dummy(&self) -> (A,B,C); }
+   | ------------------ defined here, with 3 type parameters: `A`, `B`, `C`
```

Comment:
It'd be interesting, I think, to point at the type params instead:

```
LL | trait Three<A,B,C> { fn dummy(&self) -> (A,B,C); }
   |       ----- - - -
   |       |
   |       defined here, with 3 type parameters
```

I'll check if this can be done easily.